readthedocs/api/v2/views/model_views.py

```diff
@@ -324,15 +348,15 @@ def get_queryset(self):
         own = self.request.query_params.get('own', None)
```

Comment:
What is `own` here? Could probably use a better name 

Reply:
I'm not sure why it's called `own` here, I think it refers to the VCS provider names ('github', 'gitlab' etc). We can change the variable name but the quey param name is for the API V2

Reply:
We could probably make a note on this for the APIv3 where these endpoints are not created yet: https://github.com/readthedocs/readthedocs.org/pull/7510